taskcluster/kinds/evaluate/kind.yml

```diff
@@ -126,8 +126,8 @@ tasks:
             toolchain:
                 - marian
 
-    teacher-{provider}-{dataset}-{src_locale}-{trg_locale}-{this_chunk}/{total_chunks}:
-        description: teacher evaluation for {dataset} {src_locale}-{trg_locale} {this_chunk}/{total_chunks}
+    teacher-{provider}-{dataset_sanitized}-{src_locale}-{trg_locale}-{this_chunk}:
```

Comment:
Th changes about `total_chunk` look unrelated and are likely a part of https://github.com/mozilla/firefox-translations-training/pull/606. The teacher fails in CI because W&B publisher can't parse the new labels.

Reply:
Yeah this was intentional. From the PR description:

> This PR also includes https://github.com/mozilla/firefox-translations-training/pull/606 since the CI failed due to this issue.